source/common/orca/orca_parser.cc

```diff
@@ -49,6 +49,16 @@ absl::Status tryCopyNamedMetricToOrcaLoadReport(absl::string_view metric_name, d
   return absl::OkStatus();
 }
 
+absl::Status tryCopyUtilizationToOrcaLoadReport(absl::string_view metric_name, double metric_value,
+                                                OrcaLoadReport& orca_load_report) {
+  if (metric_name.empty()) {
+    return absl::InvalidArgumentError("utilization metric key is empty.");
+  }
+
+  orca_load_report.mutable_utilization()->insert({std::string(metric_name), metric_value});
```

Comment:
I suggest adding a validation that the metric value is in [range](https://github.com/cncf/xds/blob/b4127c9b8d78b77423fd25169f05b7476b6ea932/xds/data/orca/v3/orca_load_report.proto#L38), and return an error if it isn't.